### ISSUE TITLE
Adding missing fields to ComponentAreaPath to unblock insertion PR

### ISSUE THREAD

**Issue by StellaHuang95** [microsoft/PTVS#7177]
This is from the comment that blocks our insertion PR:

For bug filing automation, currently VerifyInsertion (HashCheck and SymbolsCheck), src\ProductData\Internal\​ComponentAreaPath​.csv has been created with what is known about Components (packages in default.config, components.json and SetupPackages*.swr files with URLs)

As of September 2022 a new PR Comment will be added to your PR if some required fields in the ComponentAreaPath.csv are missing. Please update your PR with an updated ComponentAreaPath.csv with the missing information. 

Packages missing fields:

- [ ] Microsoft.CookiecutterTools.Vsix
- [ ] Microsoft.CookiecutterTools.Vsix.Resources
- [ ] Microsoft.PythonTools.BuildCore.Vsix
- [ ] Microsoft.PythonTools.Core.Templates.Vsix
- [ ] Microsoft.PythonTools.Core.Vsix
- [ ] Microsoft.PythonTools.Core.Vsix.Resources
- [ ] Microsoft.PythonTools.Debugger.VCLauncher.Vsix
- [ ] Microsoft.PythonTools.Django.Templates.Vsix
- [ ] Microsoft.PythonTools.Django.Vsix
- [ ] Microsoft.PythonTools.Django.Vsix.Resources
- [ ] Microsoft.PythonTools.NativeDevelopment.Templates.Vsix
- [ ] Microsoft.PythonTools.Profiling.Vsix
- [ ] Microsoft.PythonTools.Profiling.Vsix.Resources
- [ ] Microsoft.PythonTools.Web.Templates.Vsix
